model/manifest.go

```diff
@@ -166,6 +166,9 @@ type Manifest struct {
 
 	// Plugins can store any kind of data in Props to allow other plugins to use it.
 	Props map[string]interface{} `json:"props,omitempty" yaml:"props,omitempty"`
+
+	// RequiredConfig defines any required server configuration fields for the plugin to function properly.
```

Comment:
Can you please add a comment referencing `CheckRequiredServerConfiguration`. Something like "Use Helpers.CheckRequiredServerConfiguration` to enforce this."

Reply:
I added the documentation, thanks!